aspnetcore/whats-new/2020-12.md

```diff
@@ -10,6 +10,10 @@ Welcome to what's new in the ASP.NET Core docs from December 1, 2020 through Dec
 
 ## Blazor
 
+### New articles
+
+- [Blazor: Route precedence logic changed in Blazor apps](/dotnet/core/compatibility/aspnet-core/5.0/blazor-routing-logic-changed.md) - Document Blazor route matching precedence logic
```

Comment:
Hi @BillWagner :wave: ... 

```suggestion
- [Blazor: Route precedence logic changed in Blazor apps](/dotnet/core/compatibility/aspnet-core/5.0/blazor-routing-logic-changed) - Document Blazor route matching precedence logic
```

... and FYI ... we have the coverage for it in the Migration topic at ...

https://docs.microsoft.com/en-us/aspnet/core/migration/31-to-50?view=aspnetcore-5.0&tabs=visual-studio#changes-to-blazor-app-routing-logic-in-501

Reply:
I still think it's worth adding to the what's new page to raise awareness.

Reply:
Also note @scottaddie ... there's a **_breaking change_** to the **_change_** :smile: ... `PreferExactMatches` has to be set to `@true` for it to work ... `true` won't cut it ... it takes that as a string and :boom:. The article over there ...

https://docs.microsoft.com/en-us/dotnet/core/compatibility/aspnet-core/5.0/blazor-routing-logic-changed

... should have that update. I'm 👂 if you want me to send that in over there.

Reply:
> it's worth adding

Yes ... I was just mentioning that in passing.